lib/linter-reek.js

```diff
@@ -2,20 +2,10 @@
 
 // eslint-disable-next-line import/no-extraneous-dependencies, import/extensions
 import { CompositeDisposable } from 'atom';
+import * as helpers from 'atom-linter';
```

Comment:
When the package gets _activated_ is controlled by the `activationHook`, however the package is still _loaded_ on Atom startup. These should remain lazy loaded as before, however the deferred call in `activate` is no longer meaningful.